window_management/window_tweak.talon

```diff
@@ -0,0 +1,121 @@
+
+# show size and position information about the current window
+win show$:
+    user.win_show()
+
+# hide the window information window
+win hide$:
+    user.win_hide()
+
+# move the current window toward the center of the screen
+win move$:
```

Comment:
Why are all of these commands end-anchored? Might be useful to remove that.

Reply:
Fixed.